public/Miscellaneous/printYoutube.js

```diff
@@ -1,35 +1,78 @@
-if (!window["youtubeIFrame"]) {
+﻿if (!window["youtubeIFrame"]) {
 	window["youtubeIframe"] = true;
 
 	window.addEventListener("load", () => {
 		
 		let candidates = $("iframe");
 		for (let i = 0; i < candidates.length; i++) {
 			let iframe = candidates[i];
-			if (iframe && iframe.src && !iframe.classList.contains("youtubeIFrame")) {
-				let videoID = iframe.src.match(/^.*(youtu.be\/|v\/|e\/|u\/\w+\/|embed\/|v=)([^#\&\?]*).*/);
+iframe.id = 'qrcode'+i;
+console.log(iframe.id);
+
+let videoID = iframe.src.match(/^.*(youtu.be\/|v\/|e\/|u\/\w+\/|embed\/|v=)([^#\&\?]*).*/);
 				if(videoID) {
 					console.log(videoID);
-				}
+                    videoID = videoID[2];
+				}		
+				
 				if (iframe.src.includes("www.youtube.com") || iframe.src.includes("www.youtube-nocookie.com")) {
```

Comment:
Move this condition to be the third line of this for loop (either combined with or after the iframe if statement). We don't want this code to execute for iframes that aren't youtube videos.